CMakeTests.cmake

```diff
@@ -231,6 +231,15 @@ add_h3_test(testCellToBBoxExhaustive src/apps/testapps/testCellToBBoxExhaustive.
 add_h3_cli_test(testCliCellToLatLng "cellToLatLng -c 8928342e20fffff" "POINT(-122.5003039349 37.5012466151)")
 add_h3_cli_test(testCliLatLngToCell "latLngToCell --lat 20 --lng 123 -r 2" "824b9ffffffffff")
 add_h3_cli_test(testCliCellToBoundary "cellToBoundary -c 8928342e20fffff" "POLYGON((-122.4990471431 37.4997389893, -122.4979805011 37.5014245698, -122.4992373065 37.5029321860, -122.5015607527 37.5027541980, -122.5026273256 37.5010686174, -122.5013705214 37.4995610248, -122.4990471431 37.4997389893))")
+add_h3_cli_test(testCliGetResolution "getResolution -c 85283473fffffff" "5")
+add_h3_cli_test(testCliGetBaseCellNumber "getBaseCellNumber -c 85283473fffffff" "20")
+add_h3_cli_test(testCliStringToInt "stringToInt -c 85283473fffffff" "599686042433355775")
+add_h3_cli_test(testCliIntToString "intToString -c 599686042433355775" "85283473fffffff")
+add_h3_cli_test(testCliIsValidCell "isValidCell -c 85283473fffffff" "true")
+add_h3_cli_test(testCliIsNotValidCell "isValidCell -c 85283473ffff" "false")
+add_h3_cli_test(testCliIsResClassIII "isResClassIII -c 85283473fffffff" "true")
+add_h3_cli_test(testCliIsPentagon "isPentagon -c 85283473fffffff" "false")
+add_h3_cli_test(testCliGetIcosahedronFaces "getIcosahedronFaces -c 81743ffffffffff" "3, 8, 13, 9, 4")
```

Comment:
I'm sure we talked about this before, but it seems weird to me to inline the actual test logic into the CMake file, particularly alongside other CMake commands doing more... make-y stuff. If this is really the best way to do it, can we separate out these commands into a different `CMakeCLITests.cmake` file?

Reply:
I don't remember who introduced CLI testing in this way, actually. I am pretty sure that I *didn't*, but I might be wrong about it.

In any case, I was following the existing convention and there was no objection before on this, but I agree it feels weird.

Do you mind if I take a stab at restructuring these tests during the next PR?